pkg/ruler/manager.go

```diff
@@ -304,6 +449,38 @@ func (r *DefaultMultiTenantManager) getOrCreateNotifier(userID string) (*notifie
 	return n.notifier, nil
 }
 
+// newGRPCClient returns a new gRPC client for the address.
+func newGRPCClient(address string) (httpgrpc.HTTPClient, error) {
```

Comment:
Unfortunately we can't use the [weaveworks gRPC client](https://github.com/weaveworks/common/blob/master/httpgrpc/server/server.go#L184) since it's more middleware than client, and requirs a `ResponseWriter`.